library/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -407,13 +411,16 @@ public void takePicture() {
         mImpl.takePicture();
     }
 
-    private class CallbackBridge implements CameraViewImpl.Callback {
+    private static class CallbackBridge implements CameraViewImpl.Callback {
```

Comment:
Do you actually experience memory leaks due to this?
It seems to me CallbackBridge won't be an issue. It does hold a reference to CameraView, but it won't outlive the CameraView.